### PR TITLE
Enable to configure geofilters for each collection

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/app/modules/map-config/components/edit-layer/edit-layer.component.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ import { NGXLogger } from 'ngx-logger';
 import { Subscription } from 'rxjs';
 import { KeywordColor } from '../dialog-color-table/models';
 import { LAYER_MODE } from './models';
+import { MapGlobalFormBuilderService } from '../../services/map-global-form-builder/map-global-form-builder.service';
```
Comment (sebbousquet) [inline src/app/modules/map-config/components/edit-layer/edit-layer.component.ts:33]: Use import annotation `@service...`

File: src/app/modules/map-config/components/layers/layers.component.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ import { MapContributor } from 'arlas-web-contributors';
 import { ArlasCollaborativesearchService, ArlasColorGeneratorLoader, ArlasConfigService, ContributorBuilder } from 'arlas-wui-toolkit';
 import { Subscription } from 'rxjs';
 import { PreviewComponent } from '../preview/preview.component';
+import { MapGlobalFormBuilderService } from '../../services/map-global-form-builder/map-global-form-builder.service';
```
Comment (sebbousquet) [inline src/app/modules/map-config/components/layers/layers.component.ts:46]: Use import annotation `@service...`

File: src/app/modules/map-config/components/layers/layers.component.ts [hunk 3]
```diff
@@ -231,6 +233,32 @@ export class LayersComponent implements OnInit, OnDestroy {
...
+        });
+        // if (declaredCollections.length === 0) {
+        //   const mainCollection = this.mainFormService.getMainCollection();
+        //   const collectionParams = this.collectionService.collectionParamsMap.get(mainCollection);
+        //   mapGlobalFg.addGeoFilter(mainCollection, this.mapGlobalFormBuilder.buildRequestGeometry(
+        //     mainCollection,
+        //     collectionParams.params.centroid_path,
+        //     'intersects'
+        //   ));
+        // }
```
Comment (sebbousquet) [inline src/app/modules/map-config/components/layers/layers.component.ts]: why this part is not active ?

File: src/app/modules/map-config/services/map-init/map-init.service.ts [hunk 4]
```diff
@@ -66,13 +66,12 @@ export class MapInitService {
...
-          c.params.geometry_path,
-          c.params.id_path
-        )
-      );
+      // const mapGlobalFg = this.mainFormService.mapConfig.getGlobalFg();
+      // mapGlobalFg.addGeoFilter(collection, this.mapGlobalFormBuilder.buildRequestGeometry(
+      //   collection,
+      //   c.params.geometry_path,
+      //   'intersects'
+      // ));
```
Comment (sebbousquet) [inline src/app/modules/map-config/services/map-init/map-init.service.ts]: To remove ?

File: src/app/shared/components/config-form-group-array/config-form-group-array.component.scss [hunk 5]
```diff
@@ -25,3 +25,38 @@ div .form-group-form-fields {
         margin-right: 20px;
     }
 }
+
+@import "variables.scss";
```
Comment (sebbousquet) [inline src/app/shared/components/config-form-group-array/config-form-group-array.component.scss:29]: Move `import` at the top of file, is more understandable

Comment (MohamedHamouGisaia) [reply src/app/shared/components/config-form-group-array/config-form-group-array.component.scss]: Fixed

File: src/app/modules/map-config/components/layers/layers.component.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/modules/map-config/components/layers/layers.component.ts]: Fixed

File: src/app/modules/map-config/components/edit-layer/edit-layer.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/modules/map-config/components/edit-layer/edit-layer.component.ts]: Fixed

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**